### PR TITLE
Make product_install install Satellite 6 Beta

### PR DESCRIPTION
**Pull request by elyezer**
Add a new distribution called satellite6-beta in order to make
product_install install use the Satellite 6 Beta repository.

### CODE REVIEW

File: automation_tools/repository.py [hunk 1]
```diff
@@ -99,15 +99,22 @@ def create_custom_repos(**kwargs):
         repo_file.close()
 
 
-def enable_satellite_repos(cdn=False, disable_enabled=True):
+def enable_satellite_repos(cdn=False, beta=False, disable_enabled=True):
```
Comment (Ichimonji10) [inline automation_tools/repository.py]: Will adding a new argument in this manner break any calls to this method that use positional arguments? Or do we have no calls to this method that are positional?

File: automation_tools/repository.py [hunk 2]
```diff
@@ -99,15 +99,22 @@ def create_custom_repos(**kwargs):
...
 
-    If ``disable_enabled`` is ``True``, then this task will first disable any
-    already enabled repository, including beaker ones.
+    :param cdn: Indicates if the CDN Satellite 6 repo should be enabled or not
+    :param beta: Indicates if the Beta Satellite 6 repo should be enabled or
+        not. The Beta repo is available through the CDN and, if both ``cdn``
+        and ``beta`` are ``True``, will be used the beta repo instead the
+        stable one.
+    :param disable_enabled: If ``True``, then first will be disabled any
+        already enabled repository, including beaker ones.
```
Comment (Ichimonji10) [inline automation_tools/repository.py]: How about:

> If `True`, disable all repositories (including beaker repositories) before enabling repositories.

**Comment by Ichimonji10** [comment]
ACK pending comments

File: automation_tools/repository.py (hunk 1 above)
Comment (elyezer) [inline automation_tools/repository.py]:
```
$ git grep enable_satellite_repos
automation_tools/__init__.py:from automation_tools.repository import enable_satellite_repos
automation_tools/__init__.py:        enable_satellite_repos,
automation_tools/repository.py:def enable_satellite_repos(cdn=False, disable_enabled=True):
automation_tools/satellite5.py:from automation_tools.repository import enable_satellite_repos
automation_tools/satellite5.py:    execute(enable_satellite_repos, host=host)
fabfile.py:    enable_satellite_repos,
```

File: automation_tools/repository.py (hunk 2 above)
Comment (elyezer) [inline automation_tools/repository.py]: Thank you

File: automation_tools/repository.py [hunk 3]
```diff
@@ -99,15 +99,22 @@ def create_custom_repos(**kwargs):
...
-def enable_satellite_repos(cdn=False, disable_enabled=True):
+def enable_satellite_repos(cdn=False, beta=False, disable_enabled=True):
     """Enable repositories required to install Satellite 6
 
-    If ``disable_enabled`` is ``True``, then this task will first disable any
-    already enabled repository, including beaker ones.
+    :param cdn: Indicates if the CDN Satellite 6 repo should be enabled or not
+    :param beta: Indicates if the Beta Satellite 6 repo should be enabled or
+        not. The Beta repo is available through the CDN and, if both ``cdn``
+        and ``beta`` are ``True``, will be used the beta repo instead the
```
Comment (JacobCallahan) [inline automation_tools/repository.py]: "will be using the" or "will use the"
"instead of the"

**Comment by JacobCallahan** [comment]
ACK, pending small comments

File: automation_tools/repository.py (hunk 3 above)
Comment (elyezer) [inline automation_tools/repository.py]: Thanks, updating...

**Comment by sthirugn** [comment]
Will my install job have options for me to choose between beta and 6.0.8?

**Comment by omaciel** [comment]
@sthirugn with this change, when you choose `CDN` you'll get `6.0.8` and `BETA` will give you `6.1.0`

**Comment by sthirugn** [comment]
:+1:

**Pull request merged**